prometheus/cmd/receive_adapter/main.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"knative.dev/eventing/pkg/adapter"
+
+	prometheusadapter "knative.dev/eventing-contrib/prometheus/pkg/adapter"
+)
+
+func main() {
+	adapter.Main("prometheussource", prometheusadapter.NewEnvConfig, prometheusadapter.NewAdapter)
```

Comment:
`prometheus_source` ? or `prometheussource` ? 

we use a different naming here:
https://github.com/knative/eventing-contrib/pull/663/files#diff-49eaaa04b021b672f903ed6842e4708dR25

Reply:
Let's go with prometheussource, it's consistent with prometheussources usage in the CRD